.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -41,23 +91,51 @@ If you need to submit a pr for any of these repos during this period, you should
   </ul>
 </details>
 
-- [ ] **Enable code freeze bot** In order to enable the code freeze GitHub you need to change the line `if: github.repository_owner == 'adoptium' && false` to be `if: github.repository_owner == 'adoptium' && true` in the [code-freeze.yml](https://github.com/adoptium/.github/blob/main/.github/workflows/code-freeze.yml#L21) GitHub workflow. Please contact the PMC if you need help merging this change.
-- [ ] **Disable nightly build completely** by disabling the pipelines temporarily to free up resources and ensure no competing jobs during release week
-- [ ] Tag the build repositories (temurin-build, ci-jenkins-pipelines, jenkins-helper) for the release level tooling: github/\<repo\>/releases, "Draft new release". Choose tag: "v\<YYYY\>.\<MM\>.01" (increment 01 if needed..)
+ - [ ] **Enable code freeze bot** In order to enable the code freeze GitHub you need to change the line `if: github.repository_owner == 'adoptium' && false` to be `if: github.repository_owner == 'adoptium' && true` in the [code-freeze.yml](https://github.com/adoptium/.github/blob/main/.github/workflows/code-freeze.yml#L21) GitHub workflow. Please contact the PMC if you need help merging this change.
+ - [ ] **Freeze The Non Tagged Repositories** implement code freeze on the following repositories : 		
+	 - github-release-scripts https://github.com/adoptium/github-release-scripts
+	 - containers https://github.com/adoptium/containers
+	 - installer https://github.com/adoptium/installer
+	 - mirror-scripts https://github.com/adoptium/mirror-scripts
+
+- [ ] **Disable standard & evaluation nightly builds completely** by disabling the pipelines temporarily to free up resources and ensure no competing jobs during release week
```

Comment:
Should add a note to do this by setting defult for ENABLE_PIPELINE_SCHEDULE to false in the https://ci.adoptium.net/job/build-scripts/job/utils/job/build-pipeline-generator/, and then re-running it.

Reply:
otherwise if you just disable the project, upon a PR merge regen will re-enable it...

Reply:
Yes, the proper way is to do it from config and re-run seed job.

Last time, I did the shortcut to only disabled it from Jenkins UI, based on no new commit could be merged to "master" of ci-jenkins-pipeline during the release period.  
But since we are using release branches now, it is safer to do it from config.

Reply:
Done!